examples/tox21/keras_fcnet.py

```diff
@@ -0,0 +1,60 @@
+"""
+Script that trains multitask models on Tox21 dataset.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import numpy as np
+import keras
+from keras import Input, Model
+from keras.layers import Dense, Dropout, Multiply
+import tensorflow as tf
+
+import deepchem as dc
+
+
+def reshape_y(y):
+  vectors = []
+  for i in range(y.shape[1]):
+    vectors.append(y[:, i])
+  return vectors
+
+
+# Only for debug!
+np.random.seed(123)
+
+metric = dc.metrics.Metric(dc.metrics.roc_auc_score, np.mean)
+
+# Load Tox21 dataset
+n_features = 1024
+tox21_tasks, tox21_datasets, transformers = dc.molnet.load_tox21()
+train_dataset, valid_dataset, test_dataset = tox21_datasets
+
+inputs = Input(shape=train_dataset.X[0].shape)
+x = Dense(1000, activation='relu')(inputs)
+x = Dropout(0.25)(x)
+outputs = []
+for i in range(len(tox21_tasks)):
+  predictions = Dense(2, activation='softmax')(x)
+  outputs.append(predictions)
+
+optimizer = keras.optimizers.Adam(lr=0.001)
+model = Model(inputs=inputs, outputs=outputs)
+model.compile(
+    optimizer=optimizer,
+    loss='sparse_categorical_crossentropy',
+    metrics=['accuracy'])
+model.fit(
+    train_dataset.X,
+    reshape_y(train_dataset.y),
+    batch_size=50,
+    epochs=1,
+    verbose=False,
```

Comment:
when verbose is True it gives you all sorts of nice messages about how training is going!